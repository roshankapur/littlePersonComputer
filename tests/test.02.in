1009
1010
2009
3010
2111
1111
4300
-99999
-9999
9999