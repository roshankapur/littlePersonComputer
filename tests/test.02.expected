***Start LPC Execution***
Contents of 11: 0
***End LPC Execution***
